### PR TITLE
fix(tasks): improve task update/create callbacks

### PR DESCRIPTION
**Pull request by alexkrolick**
- fix pending state:
  - add styles to task card to match comment pending state
  - add CSS transition
- fix update logic of task assignment by refetching task to get updated status
- improve error handling of initial task create using try/catch in async/await context

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @alexkrolick has signed the CLA. Thanks for the pull request!

File: src/api/Feed.js [hunk 1]
```diff
@@ -392,15 +393,16 @@ class Feed extends Base {
             id: taskCollaboratorId,
             status: taskCollaboratorStatus,
         };
+        const handleError = (e: ElementsXhrError, code: string) => {
```
Comment (jstoffan) [inline src/api/Feed.js]: It looks like having the success/error handler wrappers inline or as instance properties is common across most other API calls. Can this stay consistent as well?

Comment (alexkrolick) [reply src/api/Feed.js]: This needs to be passed in 2 places now, rather than just the one callback. It looks like the inline callback was being wrapped.

**Pull request merged**